### PR TITLE
added a bunch more fox noises

### PR DESCRIPTION
**Pull request by hitomishirichan**
## About the PR

renault can make a bunch more legally distinct copyright free fox noises

## Why / Balance

we've been doing a lot of personal practice projects to figure out how the codebase works and stuff and this was just a whim of ours so we decided why not see of people like it or something so we're making the PR for the bit. and also it would be kinda cool to voice a fox in a video game...(for clarification, we recorded the samples ourselves with our own voice.)

## Media

- [ X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

https://github.com/space-wizards/space-station-14/assets/12144569/b0170dde-3855-437f-8120-9b7515696ddb

## Changelog

🆑 fujiwaranao
- add: Renault now has additional fox noises.

### CODE REVIEW

**Comment by exincore** [comment]
This probably needs an attributions entry at `Resources/Audio/Animals/attributions.yml`. Based on the description, I assume you voiced these samples yourself? If so, the entry might look something like...

```yml
- files:
    - fox1.ogg
    - fox2.ogg
    - fox3.ogg
    - fox4.ogg
    - fox5.ogg
    - fox6.ogg
    - fox7.ogg
    - fox8.ogg
    - fox9.ogg
    - fox10.ogg
    - fox11.ogg
    - fox12.ogg
    - fox13.ogg
    - fox14.ogg
    - fox15.ogg
    - fox16.ogg
    - fox_distort.ogg
  copyright: "Created by fujiwaranao"
  license: "CC0-1.0"
  source: "https://github.com/space-wizards/space-station-14/pull/27578"
```

Pick a license like `CC0-1.0` or `CC-BY-NC-SA-4.0` or whatever you wish, as long as it is compatible with this codebase and formatted according to [SPDX License Identifiers](https://spdx.org/licenses/). The original `fox_squeak.ogg` is `CC0-1.0`.

See [Robust Generic Attribution](https://docs.spacestation14.com/en/specifications/robust-generic-attribution.html?highlight=attributions#robust-generic-attribution).

**Comment by hitomishirichan** [comment]
> This probably needs an attributions entry at `Resources/Audio/Animals/attributions.yml`. Based on the description, I assume you voiced these samples yourself? If so, the entry might look something like...
> 
> ```yaml
> - files:
>     - fox1.ogg
>     - fox2.ogg
>     - fox3.ogg
>     - fox4.ogg
>     - fox5.ogg
>     - fox6.ogg
>     - fox7.ogg
>     - fox8.ogg
>     - fox9.ogg
>     - fox10.ogg
>     - fox11.ogg
>     - fox12.ogg
>     - fox13.ogg
>     - fox14.ogg
>     - fox15.ogg
>     - fox16.ogg
>     - fox_distort.ogg
>   copyright: "Created by fujiwaranao"
>   license: "CC0-1.0"
>   source: "https://github.com/space-wizards/space-station-14/pull/27578"
> ```
> 
> Pick a license like `CC0-1.0` or `CC-BY-NC-SA-4.0` or whatever you wish, as long as it is compatible with this codebase and formatted according to [SPDX License Identifiers](https://spdx.org/licenses/). The original `fox_squeak.ogg` is `CC0-1.0`.
> 
> See [Robust Generic Attribution](https://docs.spacestation14.com/en/specifications/robust-generic-attribution.html?highlight=attributions#robust-generic-attribution).

oh oh thank you yes we did voice this ourselves aaa we're shy anyway thank you so much for the sample file

**Comment by exincore** [comment]
Also, the changelog may need some grammatical edits.

`- add: Renault can make a bunch more legally-distinct, copyright-free fox noises.`

Strictly speaking, it does not have "significant in-game impact" to warrant a changelog entry; however, I *do* find this significantly cool, and the maintainers might not mind.

See [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html?highlight=changelog#changelog) (for reference; I am confident you have read those).

File: Resources/Audio/Animals/attributions.yml [hunk 1]
```diff
@@ -117,24 +117,45 @@
...
+    - fox10.ogg
+    - fox11.ogg
+    - fox12.ogg
+    - fox13.ogg
+    - fox14.ogg
+    - fox15.ogg
+    - fox16.ogg
+    - fox_distort.ogg
+  copyright: "Created by fujiwaranao"
+  license: "CC BY-NC-SA 4.0"
```
Comment (exincore) [inline Resources/Audio/Animals/attributions.yml]: Replace the spaces in the license with dashes as per https://spdx.org/licenses/.

Should be `  license: "CC-BY-NC-SA-4.0"`

Comment (hitomishirichan) [reply Resources/Audio/Animals/attributions.yml]: oh thank you oopsie

**Comment by hitomishirichan** [comment]
> Also, the changelog may need some grammatical edits.
> 
> `- add: Renault can make a bunch more legally-distinct, copyright-free fox noises.`
> 
> Strictly speaking, it does not have "significant in-game impact" to warrant a changelog entry; however, I _do_ find this significantly cool, and the maintainers might not mind.
> 
> See [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html?highlight=changelog#changelog) (for reference; I am confident you have read those).

We have removed the changelog entry. We had misread the portion about changelog entries in that we'd somehow gotten it mixed up and thought visual (and probably by extension audio) changes were to be logged, but we'd misunderstood.

Also, we're flattered that this has been received so positively...we didn't expect to be considered so seriously and kinda thought people were just going to laugh at us but given this reception, we're going to take some time to go through the audio files again and make sure the sound mixing is tuned to the best of our ability aaaa thank you so much

**Comment by exincore** [comment]
> We have removed the changelog entry. We had misread the portion about changelog entries in that we'd somehow gotten it mixed up and thought visual (and probably by extension audio) changes were to be logged, but we'd misunderstood.

Keep in mind I'm not a maintainer and my voice is not the be-all and end-all. I just figured while I was looking over the grammar I would give my thoughts on that. Since it is not an invisible change, I say keep the changelog; A maintainer can remove it at their discretion. :)

**Comment by hitomishirichan** [comment]
showcase video updated with the cleaned up sounds 😊

File: Resources/Audio/Animals/attributions.yml [hunk 2]
```diff
@@ -117,24 +117,42 @@
...
+    - fox6.ogg
+    - fox7.ogg
+    - fox8.ogg
+    - fox9.ogg
+    - fox10.ogg
+    - fox11.ogg
+    - fox12.ogg
+    - fox13.ogg
+    - fox14.ogg
+  copyright: "Created by fujiwaranao"
```
Comment (VasilisThePikachu) [inline Resources/Audio/Animals/attributions.yml:156]: You are a fox? (Copyright needs to be where you took the sounds from and what modifications you made etc. Source is the page you downloaded the sounds from (example freesound) check above)

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]

**Comment by mirrorcult** [comment]
kind of based

**Comment by EmoGarbage404** [comment]
it might be better to opt for some [creative-commons fox recordings](https://freesound.org/people/tivui64/sounds/587257/) just for something that sounds a bit more realistic.

**Comment by Interrobang01** [comment]
> it might be better to opt for some [creative-commons fox recordings](https://freesound.org/people/tivui64/sounds/587257/) just for something that sounds a bit more realistic. 

the sound linked there would probably be better suited for our falling-into-a-hole sound effects than our fox-petting sound effects, but yes, creative commons sound effects would in theory be more realistic (wikimedia commons probably has a lotta of fox sounds)

on the other hand, nobody actually knows what foxes really sound like (as we all know), so really anything other than demonic screaming would probably pass

**Comment by EmoGarbage404** [comment]
> on the other hand, nobody actually knows what foxes really sound like (as we all know), so really anything other than demonic screaming can pass as realistic

people do in fact know what foxes sound like, hence why i recognized that the sound effects here don't really sound that accurate to foxes. In fact, you can watch extensive footage and recordings of foxes making various sounds.

File: Resources/Audio/Animals/attributions.yml (hunk 2 above)
Comment (hitomishirichan) [reply Resources/Audio/Animals/attributions.yml:156]: yes actually we made the fox noises ourselves with our own vocal cords hehe

**Comment by hitomishirichan** [comment]
> it might be better to opt for some [creative-commons fox recordings](https://freesound.org/people/tivui64/sounds/587257/) just for something that sounds a bit more realistic.

these are vixen cries and foxes sound like this under very specific circumstances. the noses we produced replicate the kinds of noises foxes make when playing or like...being pet and stuff when more domesticated

**Comment by exincore** [comment]
> people do in fact know what foxes sound like, hence why i recognized that the sound effects here don't really sound that accurate to foxes. In fact, you can watch extensive footage and recordings of foxes making various sounds.

The sounds here are reminiscent of some recordings of *captive* foxes making various non-screaming noises, like Finnegan and others on the YouTube channel SaveAFox.

**Comment by hitomishirichan** [comment]
> I am wondering if this be added to the base `MobFox` prototype instead of just Renault?

thanks for reminding us 😊

**Comment by hitomishirichan** [comment]
> Do not use codeblock for changelog (sorry if my example misled you). Also present active voice is preferred e.g. "Renault now has additional fox noises."

eeep thanks for the suggestion we didn't realize we were dipping into the past perfect tense out of habit when writing formally

**Comment by VasilisThePikachu** [comment]
Wait... you did these yourself.... ok damn.... you win

**Review by VasilisThePikachu — APPROVED** [review]

**Comment by hitomishirichan** [comment]
OH MY GOD THANK YOU SO MUCH THIS MAKES US SO HAPPY AAAA

**Pull request merged**